tt_metal/hw/inc/ethernet/dataflow_api.h

```diff
@@ -135,7 +135,7 @@ FORCE_INLINE
 void wait_for_fd_packet() {
     // There may not be a valid cmd here, since DST router is always polling
```

Comment:
Can you remove
```
    volatile uint32_t fast_dispatch_buffer_msgs_sent;
    uint32_t reserved_3_;
    uint32_t reserved_4_;
    uint32_t reserved_5_;
```
from erisc_info_t? I think this could be a latent bug. I'm hardcoding the size of this to 32 bytes in the L1 map. So your SEMAPHORES are actually getting sent here.